src/main/java/org/apache/sysml/api/mlcontext/MLResults.java

```diff
@@ -330,9 +330,19 @@ public double getDouble(String outputName) {
 		return so.getDoubleValue();
 	}
 
+	public Object get(String outputName) {
```

Comment:
If this method is intended as a public API method, can you please add appropriate JavaDoc?